### PR TITLE
Clarify environment instructions

### LINKED ISSUE #371: Tutorial: remove multi-persona comment in Initialize an Environment section
https://radapp.dev/getting-started/tutorial/init-environment/#initialize-a-radius-environment
"This step is usually done either by an infra-admin person or a developer depending upon the setup of an enterprise."

We've gotten feedback from a couple of our usability testers that this sentence is jarring and they don't really know what to do with it. 

_________________________

User 1: "This is a non-information statement."

User 2: "This sentence seems unnecessary. if the tutorial is aimed at devs, find a home for this content elsewhere."

### PR DESCRIPTION
**Pull request by AaronCrawfis**
Closes https://github.com/project-radius/docs/issues/371

### CODE REVIEW

**Review by emily-potyraj — DISMISSED** [review]
LGTM, though I'm pinging @Reshrahim too since I know she's thought _tons_ about this verbiage in the past :)

**Comment by Reshrahim** [comment]
> LGTM, though I'm pinging @Reshrahim too since I know she's thought _tons_ about this verbiage in the past :)

If we are aiming at making the tutorial dev centric, we should remove the persona-based content at these places too to make it consistent

- https://radapp.dev/getting-started/tutorial/application/#next-step-moving-to-production-on-azure
- https://radapp.dev/getting-started/tutorial/3-tutorial-connector/

@emily-potyraj - Did we get any feedback from the users on the handoff content at above places?

**Comment by Reshrahim** [comment]
> Azure Static Web Apps: Your stage site is ready! Visit it here: https://blue-river-0786b7c1e-378.westus2.2.azurestaticapps.net

@AaronCrawfis - Has the stage site domain changed? I am not able to access.

**Comment by AaronCrawfis** [comment]
> > Azure Static Web Apps: Your stage site is ready! Visit it here: https://blue-river-0786b7c1e-378.westus2.2.azurestaticapps.net
> 
> @AaronCrawfis - Has the stage site domain changed? I am not able to access.

Yeah I switched it to a new one when debugging a timeout issue with Static Web Apps. Turned out to be a platform issue that the SWA team resolved on Friday, so I've reverted to the old site with permissions already configured.

**Comment by emily-potyraj** [comment]
> > > Azure Static Web Apps: Your stage site is ready! Visit it here: https://blue-river-0786b7c1e-378.westus2.2.azurestaticapps.net
> > 
> > 
> > @AaronCrawfis - Has the stage site domain changed? I am not able to access.
> 
> Yeah I switched it to a new one when debugging a timeout issue with Static Web Apps. Turned out to be a platform issue that the SWA team resolved on Friday, so I've reverted to the old site with permissions already configured.

FWIW I'm not currently able to access the staging site @AaronCrawfis. Have to pull it up locally :)

**Comment by AaronCrawfis** [comment]
> FWIW I'm not currently able to access the staging site @AaronCrawfis. Have to pull it up locally :)

Are you using the old `blue-river` link from above, or the updated `wonderful-plane`? This is the correct one: https://wonderful-plant-020417a1e-378.westus2.azurestaticapps.net/

File: docs/content/getting-started/tutorial/2-tutorial-app/index.md [hunk 1]
```diff
@@ -159,6 +159,6 @@ Now you are ready to deploy the application for the first time:
 
 ## Next step: Moving to production on Azure
 
-This step has shown how a developer can test the application with a Mongo container. When moving to production, the dev may now want to handoff the deployment to the operations team for deployments to other environments. The operator can now setup a Radius environment with Azure cloud provider and use the same app, but now connect it to an Azure CosmosDB using the Mongo connector. The next step will show how the connector can be updated to use a CosmosDB instead of a Mongo container:
+When moving to production, you may want to swap your locally running container for an Azure CosmosDB for your Mongo database. The next step will show you how to swap your connector infrastructure to an Azure resource, leveraging the Azure [cloud provider]({{< ref providers >}}).
```
Comment (emily-potyraj) [inline docs/content/getting-started/tutorial/2-tutorial-app/index.md:162]: Nice. I think this change makes it a lot simpler and readable.

**Review by emily-potyraj — APPROVED** [review]

**Pull request merged**